internal/cmd/connect/command_custom_plugin_create.go

```diff
@@ -0,0 +1,83 @@
+package connect
+
+import (
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/spf13/cobra"
+	"path/filepath"
+)
+
+type pluginCreateOut struct {
+	Id         string `human:"ID" serialized:"id"`
+	Name       string `human:"Name" serialized:"name"`
+	ErrorTrace string `human:"Error Trace,omitempty" serialized:"error_trace,omitempty"`
+}
+
+func (c *customPluginCommand) newCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "create <name>",
+		Short:       "Create a custom connector plugin.",
+		Args:        cobra.ExactArgs(1),
+		RunE:        c.createCustomPlugin,
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Create a custom connector plugin ",
+				Code: "confluent connect custom-plugin create plugin_name --config-file config.json",
+			},
+		),
+	}
+
+	cmd.Flags().String("plugin-file", "", "ZIP/JAR plugin file.")
+	cmd.Flags().String("description", "", "description of plugin")
+	cmd.Flags().String("documentation-link", "", "document link of plugin")
+	cmd.Flags().String("connector-class", "", "connector class of custom plugin")
+	cmd.Flags().String("connector-type", "", "connector type of custom plugin")
+	cmd.Flags().String("sensitive-properties", "", "sensitive properties config of custom plugin")
+
+	cobra.CheckErr(cmd.MarkFlagRequired("connector-class"))
+	cobra.CheckErr(cmd.MarkFlagRequired("connector-type"))
+
+	cobra.CheckErr(cmd.MarkFlagFilename("plugin-file", "zip", "jar"))
+	cobra.CheckErr(cmd.MarkFlagRequired("plugin-file"))
```

Comment:
Please list all required flags together, in the same order that they're defined in. Take a look at some of our other commands for examples.